scripts/vfx/build_openvdb.sh

```diff
@@ -30,7 +30,7 @@ EOF
 
 fi
 
-if [[ $ASWF_IMATH_VERSION == 3* ]]; then
+if [[ $ASWF_IMATH_VERSION == 3* && $ASWF_OPENVDB_VERSION == 8* ]]; then
```

Comment:
Should there be a comment that this is / was a temporary fix to build OpenVDB 8 with Imath 3, and is now mostly obsolete since OpenVDB 9 is included in 2022 image?